packages/react/src/components/uiComponents.tsx

```diff
@@ -41,7 +41,7 @@ import { withClerk } from './withClerk';
 // });
 
 // Portal.displayName = 'ClerkPortal';
-class Portal extends React.PureComponent<MountProps, {}> {
+class Portal extends React.PureComponent<MountProps, Record<string, never>> {
```

Comment:
Just confirming if this was needed for the PR, or just a minor improvement

Reply:
minor improvement for eslint